### PR TITLE
link back to activity from congrats

### PR DESCRIPTION
**Pull request by Erin007**
Adds a "Back to activity" link to /congrats with a new `LargeChevronLink` component. 

non-RTL:
back to actvity

RTL:
back to activity rtl

(The fonts are the same size in both - just how it came out in the screenshot). 

Since /congrats is currently not linked to from anywhere, I tested by temporarily linking to congrats from /home and checking that the link works as expected.  If you type in the url to congrats directly, the "Back to activity" link just refreshes the page. 

This PR also includes a few changes to better support layout resizing.

### CODE REVIEW

File: apps/src/templates/Congrats.jsx [hunk 1]
```diff
@@ -10,6 +10,14 @@ import { Provider } from 'react-redux';
 import { combineReducers, createStore } from 'redux';
 import responsiveRedux from '../code-studio/responsiveRedux';
 
+const styles = {
+  container: {
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    maxWidth: 970
```
Comment (breville) [inline apps/src/templates/Congrats.jsx]: we could use the constant here?

File: apps/src/templates/LargeChevronLink.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    isRtl: PropTypes.bool.isRequired,
+  };
+
+  render() {
+    const { link, linkText, isRtl }= this.props;
+    const localeStyle = isRtl ? styles.right : styles.left;
+    const icon = isRtl ? "chevron-right" : "chevron-left";
+
+    return (
+      <div style={[styles.linkBox, localeStyle]}>
```
Comment (breville) [inline apps/src/templates/LargeChevronLink.jsx:52]: Have we settled on when to use arrays and when to use es6 object spread (`...`)?

Comment (Erin007) [reply apps/src/templates/LargeChevronLink.jsx:52]: There are many examples of both in the codebase. I don't know of a reason to feel strongly in favor of one or the other.

**Comment by tanyaparker** [comment]
I think if you just go to /congrats directly (not from a hoc activity) we shouldn't show the link at all.

**Comment by Erin007** [comment]
Good idea, Tanya! #19227

**Pull request merged**